### ISSUE TITLE
Splash Home Page HTML Support?

### ISSUE THREAD

**Issue by loktah** [rdyar/LabWOES#53]
We have tried to find a theme that is compatible within the ROES client however everything so far ends up showing a blue background. Does anyone know what is actually supported within the window? I need something nice looking that we can make to match our brand but cannot find anything that works.

**Comment by rdyar** [comment]
I have tried to, and took a stab at making my own css framework but gave up.
I think that there is a new way coming that will be html5 compliant - then it will be much better.

I looked at bay photos splash page which looks great - it is one giant image and then uses an html image map to make the links.

My splash page is in a repo here: https://github.com/rdyar/plpsplash - it looks weird in a browser, different than in roes.

it is awesome having it in a repo - the html parts I can update in a couple clicks. Once roes allows html5 it will be much easier to maintain everything as html instead of images.

**Comment by loktah** [comment]
Yeah, any CSS we used would always mess up. Well I hope they hurry up with the HTML5 support because you really can't find any themes these days that are basic html only that I know of for a base. 

Yeah, Bay Photos thats how I ended up doing ours however it was a nightmare trying to get it all to work and is not an easy task for simple updates. I looked at your and wow that looks nice! Im impressed you got it to look correct and not mess up.

**Comment by rdyar** [comment]
Did you see the ROES 9.5 notes today? webview is awesome, I just had our test launch updated to it and loaded in a website and it looks perfect, links to products work fine. This will be a huge improvement. Combine this with GH pages and you will be able to edit things on GH and they will update in roes no problem.

**Comment by loktah** [comment]
Hey Ron,
I got the press release however our test client and live client are still showing the old version which does not allow those features. I tried reaching out to Softworks to see why mine won't update but no response yet.

**Comment by loktah** [comment]
Funny how they would update it the day after I was working on trying to do the Webview stuff again ha!

**Comment by dpl-justin** [comment]
It is an opt-in upgrade. Digital Pro Lab just requested the update to v9.5 on our Test Client only. We don’t typically update our Live Clients in the busy season just in case there are any unforeseen issues/bugs. But the Test Client will allow us to test now and be ready to implement just as soon as our busy season dies down a bit. Given the holiday weekend, you may have to wait a few days for Softworks to make the update to your launch(s).

-Amanda

**Comment by loktah** [comment]
Hey Ron,
Well I got both of our clients updated and yes your right its awesome and solves the majority of the issue I could never solve. Big improvment!
